packages/components/src/components/buttons/TooltipButton/TooltipButton.stories.tsx

```diff
@@ -1,5 +1,8 @@
 import { Meta, StoryObj } from '@storybook/react';
 import { TooltipButton as TooltipButtonComponent } from './TooltipButton';
+import { getFramePropsStory } from '../../common/frameProps';
+import { allowedButtonFrameProps } from '../Button/Button';
```

Comment:
I'm not sure if using allowed frame props from Button is a good idea. I would rather strictly separate these props to each component so I would add `allowedTooltipButtonFrameProps` to `TooltipButton`. 
WDYT?

Reply:
TooltipButton has the same props as `Button`, except `tooltipContent`:
```
export interface TooltipButtonProps extends ButtonProps {
    tooltipContent: ReactNode;
}
```
I found that ok because `TooltipButtonProps` is defined exactly like this ("...extends ButtonProps"), and I wanted to avoid repeating code. But I can add `allowedTooltipButtonFrameProps` to `TooltipButton` anyway. I just considered that as unnecessary.